relayer/src/config.rs

```diff
@@ -322,8 +318,7 @@ pub struct ChainConfig {
     pub max_tx_size: MaxTxSize,
     #[serde(default = "default::clock_drift", with = "humantime_serde")]
     pub clock_drift: Duration,
-    #[serde(default = "default::trusting_period", with = "humantime_serde")]
```

Comment:
Removing the `serde(with)` attribute means we cannot express the trusting period as a human readable string, eg. `14days`. Let's find a way to re-enable it :)

Reply:
Done! Thanks for reviewing, @romac!